spec/sensitive_data/types/credit_card_spec.rb

```diff
@@ -84,4 +84,48 @@ def validations(cards)
       specify { expect(mask).to eq value }
     end
   end
+
+  describe 'pattern matching' do
+    shared_examples_for 'a pattern matcher' do
+      context 'valid pattern' do
+        it 'should match' do
+          expect(subject.match(valid_card)).to_not be_nil
```

Comment:
I think we should test the expected value, not only its presence.
You can keep the shared example and simply define the expected value in each context.